package.json

```diff
@@ -7,7 +7,7 @@
     "npm": "6.4.1"
   },
   "scripts": {
-    "console": "node console.js",
+    "console": "node --experimental-repl-await console.js",
```

Comment:
Why are you using this? It's for using promises in the console. Am I right? 
Be careful when changing the scripts of the package.json. Remember that other members of the team will use it, and we are changing the commands. So we have to think if it's some tool we are using or if it is a tool the team will use.